torchci/yarn.lock

```diff
@@ -1330,6 +1330,13 @@
   resolved "https://registry.npmjs.org/@bcoe/v8-coverage/-/v8-coverage-0.2.3.tgz"
   integrity sha512-0hYQ8SB4Db5zvZB4axdMHGwEaQjkZzFjQiN9LVYvIFB2nSUHW9tYpxWriPrWDASIxiaXax83REcLxuSdnGPZtw==
 
+"@clickhouse/client@^0.1.0":
```

Comment:
Are there any benefit in using clickhouse client here instead of their query API https://clickhouse.com/docs/en/get-started/query-endpoints?  The latter closely matches what we have in Rockset?

Reply:
My main concern with their query endpoints is that I haven't been able to figure out how to do version control yet.  Specifically, I haven't found an API endpoint that lets you download a saved query

Reply:
Yeah, let's bring this question to ClickHouse as this is a pretty important feature IMO.  It allows the testing preview of new version of the query on PR.